packages/node/test/stacktrace.test.ts

```diff
@@ -294,4 +294,88 @@ describe('Stack parsing', () => {
       },
     ]);
   });
+
+  test('parses with async frames', () => {
+    // https://github.com/getsentry/sentry-javascript/issues/4692#issuecomment-1063835795
+    const err: { [key: string]: any } = {};
```

Comment:
Why this over a `new Error`?

Reply:
I just blindly copied from the tests above it. I'm guessing it's because `Error` requires `name` and `message` properties.

Reply:
Name and message shouldn’t matter because we only check frames from `parseStackFrames`? Feels weird to cast the type as error.

Reply:
Changed all the tests to use `new Error()`